### ISSUE TITLE
Can I get currently active tab's URL using this package?

### ISSUE THREAD

**Issue by lpSharif** [jarrodcolburn/chromeapi#2]
I am looking for a way to get current active Tab's URL. So can this package help me to achieve that?

**Comment by jarrodcolburn** [comment]
yes, but just to clarify, in a Chrome Extension.

**Comment by jarrodcolburn** [comment]
also, I think I changed the API here in the repo, but didn't push to pub.dev

**Comment by lpSharif** [comment]
I see. Thanks for your time

**Comment by jarrodcolburn** [comment]
Are you trying to do a Chrome Extension?

**Comment by jarrodcolburn** [comment]
I'm gonna try to publish the last one I made to the Chrome Web Extension store. Don't know if it'll get approved right away this is my first time trying to publish an extension, and I never have much luck getting my stuff through the web stores the first time. And I'm trying to skip over all that policy garbage, since i don't really consider this a true Chrome extension, more just a perfect concept.

If you're serious about wanting to build a flutter Chrome extension, I can probably set up a video chat and walk you through what I did pretty quickly. I started an options page which was going to allow the user to select default styles, color foreground and background. But I didn't complete the feature, I think I need to also wrap chrome.storage and build the Flutter components.

Also,If you want to try to collaborate with some people on a chrome extension, maybe check out the Flutter Discord "collaboration". See if anyone in there wants to work on a Chrome extension. I've been trying to be better about Getting in the Flutter community myself. I'm trying to avoid rewriting the home chrome API by hand when I'm pretty sure it could be done through automation. And I'm a little unsure what the best way to wrap the crime API is, as there's some differences between how JavaScript and Dart treat libraries, semantically and otherwise. Also I'm a little unsure about the transition that's going on in the dark JS ecosystem as they prepare for wasm. I think JS `static interop` is the way word with things like this, but Not positive or if my implementation of it is correct
